<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="ClientStack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_txt_box">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>210</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pass_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_txt_box">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>150</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="id_lbl">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="register_btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>20</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Don't have an account?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="request_btn">
      <property name="geometry">
       <rect>
        <x>374</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_btn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>35</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="beg_txt_box"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="end_txt_box"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Date_txt_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>New Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_client_history_btn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>View History</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Past Trips</string>
      </property>
     </widget>
     <widget class="QPushButton" name="home_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Trip</string>
      </property>
     </widget>
     <widget class="QListView" name="past_trips">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>611</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="current_trip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>No current trip</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="home_btn_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Trip</string>
      </property>
     </widget>
     <widget class="QLabel" name="current_trip_display">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>561</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>No current trip</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="pass_label_2">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_txt_box_2">
      <property name="geometry">
       <rect>
        <x>242</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="id_lbl_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_txt_box_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="pas_rentry_txt_box">
      <property name="geometry">
       <rect>
        <x>242</x>
        <y>180</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pass_label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Re-enter Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_txt_box">
      <property name="geometry">
       <rect>
        <x>242</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="register_btn_add">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
